x-pack/plugins/reporting/export_types/csv/server/lib/check_cells_for_formulas.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as _ from 'lodash';
+
+const formulaValues = ['=', '+', '-', '@'];
+
+interface IFlattened {
+  [header: string]: string;
+}
+
```

Comment:
Core check is here. We get the row at this point as a flattened object. Keys === headers, values their contents.